UI/PlayerListWnd.cpp

```diff
@@ -593,12 +593,12 @@ std::set<int> PlayerListWnd::SelectedPlayerIDs() const {
     return retval;
 }
 
-void PlayerListWnd::HandlePlayerStatusUpdate(Message::PlayerStatus player_status, int about_player_id) {
+void PlayerListWnd::HandleEmpireStatusUpdate(Message::PlayerStatus player_status, int about_empire_id) {
     for (auto& row : *m_player_list) {
         if (PlayerRow* player_row = dynamic_cast<PlayerRow*>(row.get())) {
-            if (about_player_id == Networking::INVALID_PLAYER_ID) {
+            if (about_empire_id == ALL_EMPIRES) {
```

Comment:
Does this make sense? If a status update comes in for `ALL_EMPIRES`, is that likely to apply to all players, or none, or just the non-empire players...?

Reply:
Fixed. Changed to warning as it isn't client error to get such value.